### ISSUE TITLE
I get that extrapolation cant be removed, but..

### ISSUE THREAD

**Issue by Jacob-1998** [haxball/haxball-issues#1030]
This is because its already been added, so if you remove it people can hack it because the example has already been set.

However if you removed Haxball, and uploaded it again to the internet, without extrapolation this time, how would people be able to use extrapolation? Wouldn't it be impossible, just like it was impossible before you added extrapolation the first time?

If you do this, I know its a lot of work because bugs might appear again like when you first started HTML and its a lot of work, but in the long term I think its worth the investment of your time. Right now we are in lockdown because of coronavirus anyway, so maybe you have more free time these days.

So do this, and add the Input Delay to match the Flash version so the physics match and the gameplay matches Flash, because futsal is taking over this game, due to boring physics in normal Haxball, whereas on Flash for 7 years standard Haxball dominated and these niche game modes didn't take over, because standard Haxball on Flash physics was superior.

If you can do both these things, I will compensate you with good money, 10.000 US dollars to be exact, for these two feature requests, is my offer. Sound good?

**Comment by spr1907** [comment]
> However if you removed Haxball, and uploaded it again to the internet, without extrapolation this time, how would people be able to use extrapolation?

Some people have current codes of the game so if he uploads non extrapolation version people will just copy paste extrapolation codes into original one and it will work same as in past.

**Comment by saviola777** [comment]
I guess he's thinking of a complete rewrite which would make the current code incompatible with the game.

**Comment by Jacob-1998** [comment]
> I guess he's thinking of a complete rewrite which would make the current code incompatible with the game.

Yes. Basro is in quarantine anyway, so unless he's working from home this is a great opportunity to fix Haxball and Im even paying him for it, even though its more in his interest to fix his game than me. I simply miss Haxball and want to enjoy it again, he actually profits more from having a better product.
